scene/resources/font.cpp

```diff
@@ -495,6 +496,38 @@ Size2 Font::get_string_size(const String &p_string) const {
 
 	return Size2(w, get_height());
 }
+
+Size2 Font::get_wordwrap_string_size(const String &p_string, float p_with) const {
```

Comment:
Should this be `p_width`? I think the `d` is missing.

Reply:
true, i'll check that